BGK.Data/Repositories/ClientRepository.cs

```diff
@@ -41,12 +41,26 @@ public bool DeleteClientById(int id)
             return true;
         }
 
+        public bool RecoveryClientById(int id)
+        {
+            var res = _db.Client.FirstOrDefault(x => x.Id == id);
+
+            res.IsDeleted = true;
+            _db.SaveChanges();
+            return true;
+        }
+
         public bool ChangePasswordClient(Client newItem)
         {
             Client item = GetClientById(newItem.Id);
             item.Password = newItem.Password;
             _db.SaveChanges();
             return true;
         }
+
+        public List<Client> GetClient()
+        {
+            return _db.Client.Where(x => !x.IsDeleted).ToList();
+        }
```

Comment:
please apply the same fixes in this class as well

Reply:
Done